regimes/mx/invoice_validator.go

```diff
@@ -106,6 +121,20 @@ func (v *invoiceValidator) validLine(value interface{}) error {
 			validation.Required,
 			validation.Skip, // Prevents each tax's `ValidateWithContext` function from being called again.
 		),
+		validation.Field(&line.Item, validation.By(v.validItem)),
+	)
+}
+
+func (v *invoiceValidator) validItem(value interface{}) error {
```

Comment:
This one was a bit tricky. We were already validating the Item in the MX regime separately, however I cannot add these validations there because I need to call to the `IsMabeSupplier` function with the invoice as a parameter. In the Item standalone validation, naturally, we don't have access to invoice. That's why I implemented this particular validation as part of the invoice.

Suggestion on how to do this differently are more than welcome.